Makefile

```diff
@@ -39,28 +39,32 @@ docs.json: common.yaml omni.yaml ## Generate and validate docs.json from multipl
 		common.yaml \
 		talos-v1.10.yaml \
 		omni.yaml \
+		kubernetes-guides-v1.10.yaml \
```

Comment:
Let's start by not versioning Kubernetes guides. We'd probably have to version them with each version of Kubernetes, not each version of Talos. But for now we can try leaving off a version.